cobigen/cobigen-javaplugin-parent/cobigen-javaplugin/src/main/java/com/devonfw/cobigen/javaplugin/merger/JavaMerger.java

```diff
@@ -68,7 +69,7 @@ public String merge(File base, String patch, String targetCharset) throws MergeE
         try (FileInputStream stream = new FileInputStream(base);
             BufferedInputStream bis = new BufferedInputStream(stream);
             InputStreamReader reader = new InputStreamReader(bis, targetCharset)) {
-            lineDelimiter = determineLineDelimiter(bis, reader);
+            lineDelimiter = SystemUtil.determineLineDelimiter(bis, reader);
```

Comment:
Have to change this to determineLineDelimiter(Path)

Reply:
Adjusted now.